src/app/services/editor/editor-service.service.ts

```diff
@@ -59,7 +59,7 @@ export class EditorService {
   }
 
   public getSchemaTypes() : string[] {
-    return ["boolean", "date", "dateTime", "double", "duration", "float", "integer", "long", "string", "time"];
+    return ["boolean", "date", "dateTime", "double", "duration", "float", "integer", "long", "string", "time", "object", "enum"];
   }
 
   public getComplexSchemaTypes() : string[] {
```

Comment:
Now that I think about it, we can/should probably just get rid of this method.

Reply:
why you say get rid of it I'm assuming make it a constant instead of using a getter that returns a constant array?

Reply:
can make this change if you'd like

Reply:
It exists on the other service you created, which is probably a more appropriate place IMO.